static/js/validatorRewards.js

```diff
@@ -7,6 +7,17 @@ var subsTable = null
 // let validators = []
 
 function create_typeahead(input_container) {
+  function prepare(query, settings) {
+    settings.url = settings.url.replace("%QUERY", encodeURIComponent(query))
+    settings.beforeSend = function (jqXHR) {
```

Comment:
This code snippet appears on multiple locations. Maybe make it a function that wraps this, like you did with waitForTurnstileToken